pages/blog/announcing-graphql-yoga-2.mdx

```diff
@@ -0,0 +1,293 @@
+    useSentry() // report unexpected errors to sentry
+  ]
+})
+
+// Start the server and explore http://localhost:4000/graphql
+server.start()
+```
+
+The Envelop Plugin currently [proposes more than 35+ plugins](https://www.envelop.dev/plugins) covering most of the standard GraphQL APIs features you need in production.
+
+Ultimately, you can [develop custom Envelop plugins](https://www.envelop.dev/docs/plugins/custom-plugin) to create reusable behaviors that hook on to the GraphQL lifecycle.
+
+<p>&nbsp;</p>
+
+### Production-ready
+
+GraphQL Yoga v2 has been built in production for production usage.
+
+Built-in real-world conditions within our projects (ex: [GraphQL Mesh](https://graphql-mesh.com/)) and with some of [our clients](https://www.the-guild.dev/services), performance was highly prioritised.
+The core of Yoga is as performant as possible, and we continuously keep track and improve it.
+
+Also, the Yoga V2 repository runs performance checks on every commit and Pull Request, so we can always capture any performance regression.
+
+We continue our effort of pushing GraphQL Yoga to more production environments with the imminent release of [Redwood 1.0](https://redwoodjs.com/) that will use Yoga 2.0 as the default GraphQL server.
+
+<p>&nbsp;</p>
+
+### A standards-compliant GraphQL Server
+
+In the same way, as TypeScript aims to stay aligned with ECMAScript, GraphQL Yoga is based on several official and recognized specs:
+
+- [GraphQL-spec](https://spec.graphql.org/October2021/), [GraphQL-over-HTTP](https://github.com/graphql/graphql-over-http): guarantees your GraphQL API to work with all existing GraphQL clients (Apollo, Relay, URQL, and more).
+- [GraphQL-Multipart-Request](https://github.com/jaydenseric/graphql-multipart-request-spec): enables great File uploads support.
+- [W3C Fetch API](https://fetch.spec.whatwg.org/): we embrace the future of Node.js and provide the same developer experience on all platforms.
+
+### GraphQL Features from the future
+
+Yoga v2 supports some experimental GraphQL features such as [`@defer` and `@stream`](https://github.com/graphql/graphql-js/pull/2839), allowing you to get a taste of the future of GraphQL (with compatible clients such as [URQL](https://formidable.com/open-source/urql/docs/comparison/#core-features)).
+
+Also, thanks to the Envelop plugin system, Yoga v2 can also act as `"Babel for GraphQL"`, giving you the option to use features that are not yet in the GraphQL spec but are very useful in production today, like defer and stream and [OneOf](https://www.envelop.dev/docs/guides/using-graphql-features-from-the-future).
+
+<p>&nbsp;</p>
+
+## Get started with Yoga v2
+
+Yoga v2 provides the best GraphQL experience while giving you the freedom to use your preferred stack and tools.
+
+**Want to try it?** Give [our brand new tutorial](https://www.graphql-yoga.com/tutorial/basic/00-introduction) a try! It will guide you in building a full-featured, modern API with GraphQL Yoga.
```

Comment:
```suggestion
### Get started from scratch with our new tutorial ...

**Want to try it?** Give [our brand new tutorial](https://www.graphql-yoga.com/tutorial/basic/00-introduction) a try! It will guide you in building a full-featured, modern API with GraphQL Yoga.
```